constants/appLinks.ts

```diff
@@ -77,6 +77,14 @@ const TEXTBOOK_DEMO_START_LEARNING: GeneralLink = {
   }
 }
 
+const TEXTBOOK_DEMO_INTRODUCTION_COURSE_START_LEARNING: GeneralLink = {
+  url: 'https://project-platypus-default.apis-dev-5dd5718798d097eccc65fac4e78a33ce-0000.us-south.containers.appdomain.cloud/course/learning-states-and-qubits/introduction',
+  label: 'Start learning',
+  segment: {
+    action: 'introduction-course > header > start-learning'
```

Comment:
This CTA point to the same page than the first link on the learning paths section, so they should have the same text at the end of the action. Replace `introduction-course > header > start-learning` with `introduction-course > header > introduction`